install-dependencies.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Script that installs the dependencies needed to build and run Differential Dataflow
+
+echo "Installing dependencies for DDlog"
+echo "This script should be invoked with '. ./install-dependencies.sh' to set up the environment properly"
+
+case "$OSTYPE" in
+    linux*) ;;
+    *) echo "Unhandled operating system $OSTYPE"; exit 1;;
+esac
+
+
+echo "Installing Haskell"
```

Comment:
There is a script to install Haskell in `.travis/install-stack.sh`, and it works on both Linux and MacOS.  Shall we call that instead?

Reply:
The travis script depends on a lot of Travis-related stuff. 
This command-line is also very short, much shorter than your script.
I should investigate how to do it for the Mac, but I can't really test.

Reply:
There's nothing travis-related in the script it literally just checks OS type and runs `curl`. Despite its name, `travis_retry` is declared in the same file and simply retries the command several times.